deployment/cdk/opensearch-service-migration/lib/migration-assistance-stack.ts

```diff
@@ -180,6 +181,7 @@ export class MigrationAssistanceStack extends Stack {
         const sharedLogsSG = new SecurityGroup(this, 'sharedLogsSG', {
             vpc: props.vpc,
             allowAllOutbound: false,
+            allowAllIpv6Outbound: false,
```

Comment:
Have we verified Ipv6 gets used and does not break the solution?

Reply:
It does seem to break the solution, or at least things are not working correctly with this PR, I've reverted all IPv6 related changes.

Reply:
Fixed that issue, fresh deployment of this PR looks good
- https://github.com/opensearch-project/opensearch-migrations/pull/1153